source/community/release_notes/quince.rst

```diff
@@ -15,30 +13,163 @@ These are the release notes for the Quince release, the 17th community release o
+
 
 Instructor Experiences
 **********************
 
+* `Copy and Paste For Course Content <https://docs.openedx.org/en/open-release-quince.master/educators/how-tos/copy_paste_course_content.html>`__
 
 Administrators & Operators
 **************************
 
+New Learner Home Page 
+==================================
+
+No longer experimental, the new Learner Home has many of the same features as the old learner dashboard, with some extended functionality and 
+performance enhancements.   
+
+* The Learner Home is now built with Paragon, the Open edX design pattern library. It is accessible and easy to style with brand colors. 
+* Course cards show the course thumbnail, information about the course, and the ability to upgrade to a paid track or view/begin a course (if applicable). Further course actions (e.g. unenroll, email opt-out settings, and social media share) have been moved to the menu/triple dot icon on the course card.
+* Clicking the “Refine” button opens options to filter by course status or sort either by most recent enrollment (default) or title.
+
+.. image:: /_images/community/release_notes/palm/new_learner_home_filtering.png
+    :alt: The Refine pop-up with Course State and Sort options
+
+* All of a user’s courses are fetched on page load. To make the page manageable, we paginate that list, showing 25 courses at a time. To view other courses, a user should page through their list of courses using the pagination controls at the bottom of the course list.
+
+.. image:: /_images/community/release_notes/palm/new_learner_home_pagination.png
+    :alt: Next, previous and page number buttons appear below the list of courses
+
+* Site staff can now masquerade as users on the platform by typing a username or email in the “View as” box and hitting submit. This is designed to be “view only” so change actions (e.g. enroll, unenroll, selecting a session) are blocked in this view.
+* There is a complete list of changes in the `Open edX wiki <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2022/11/21/3584589831/2U+New+Learner+Home+Page#Comparison-with-old-experience>`_. 
+* `feat: feature flag to disable Advanced Settings <https://github.com/openedx/edx-platform/pull/32015>`__
+* `Tutor Credentials Plugin offers an admin panel where user can do configurations for the certificates of courses and programs. <https://github.com/overhangio/tutor-credentials>`__
+
+
+New Settings:
+
+* `EXAMS_DASHBOARD_MICROFRONTEND_URL <https://github.com/openedx/edx-platform/blob/d3d203cbac609adf23a6a8f003731fef12bd1ea1/lms/envs/common.py#L4992>`__
+* `ELASTIC_SEARCH_INDEX_PREFIX <https://github.com/openedx/edx-platform/blob/d3d203cbac609adf23a6a8f003731fef12bd1ea1/lms/envs/common.py#L1410>`__
+
+New Waffle Flags:
+
+* `CREATE_COURSE_WITH_DEFAULT_ENROLLMENT_START_DATS <https://github.com/openedx/edxplatform/blob/d3d203cbac609adf23a6a8f003731fef12bd1ea1/xmodule/course_block.py#L61>`__ 
```

Comment:
What's with the capital S at the end of all lines? It looks like a search and replace gone wrong. E.g:

```
['LOG_ERROR'S 
SEND_CERTIFICATE_REVOKED_SIGNAS
 content_tagging.autS
Etc.
```

Reply:
Found the smoking gun in my Neovim command history:

  1,$s/\S</\S </

🤦 

New revision incoming. Thanks for spotting this.

Reply:
it would never have happened with emacs

#justsayin

Reply:
You're right! I would have been so crippled by RSI that I would have been unable to make this error 😛